pkgs/by-name/ha/hanko/package.nix

```
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub, installShellFiles, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hanko";
+  version = "0.5.2";
+
+  src = fetchFromGitHub {
+    owner = "SRv6d";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-gytnUta/sNa8vJMvVS/DMwRc4R/8rmOnEfBKEjRpfGs=";
+  };
+  cargoHash = "sha256-rlKFS1ppUoyZ1X2PeG8tEB44Ysqi7kCLqR+ECVIdPjs=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installManPage assets/manpages/*.1
+
+    installShellCompletion assets/completions/hanko.bash
+    installShellCompletion assets/completions/_hanko
+    installShellCompletion assets/completions/hanko.fish
+  '';
+
+  meta = with lib; {
+    description = "Keeps your Git allowed signers file up to date with signing keys configured on software development platforms like GitHub and GitLab";
```

Descriptions should be short, concise phrases that fit into one sentence. More description text could always be put into `longDescription` instead.

```suggestion
    description = "Keeps your Git allowed signers file up to date";
```